### ISSUE TITLE
not a valid add-on repository

### ISSUE THREAD

**Issue by maxxnet2** [davidusb-geek/emhass#116]
**Describe the bug**
https://github.com/davidusb-geek/emhass is not a valid add-on repository

**To Reproduce**
Add the repository to HA

**Expected behavior**
Add the repository to HA

**Screenshots**
![emhass](https://github.com/davidusb-geek/emhass/assets/95687133/abe3fc42-a0a6-4837-97d9-ea32deb851b9)
![image](https://github.com/davidusb-geek/emhass/assets/95687133/249cc514-96fe-4a44-8cb1-c39e991cdd6d)

**Home Assistant installation type**
 - Home Assistant OS

**Your hardware**
- OS: HA OS
- Architecture: amd64

**EMHASS installation type**
 - Add-on

**Additional context**

**Comment by gieljnssns** [comment]
I think you have to use this one. 
https://github.com/davidusb-geek/emhass-add-on

**Comment by davidusb-geek** [comment]
> I think you have to use this one. https://github.com/davidusb-geek/emhass-add-on

Exactly, you are not using the correct URL for the add-on repository.
Closing as that will solve the issue.
